sdk/tests/conformance/extensions/webgl-multi-draw.html

```diff
@@ -1005,17 +986,46 @@
     );
   }
 
-  await compositingTestFn(drawArrays);             // sanity check
-  await compositingTestFn(drawElements);           // sanity check
+  await testFn(drawArrays);             // sanity check
+  await testFn(drawElements);           // sanity check
+
+  await testFn(multiDrawArraysWEBGL);
```

Comment:
These have to be short-circuited at the top level in order for the test to pass when the extension isn't available, because the invalid vertex attrib tests expect the drawFn to have an effect.

Reply:
Fixed in my revision to the PR.